### ISSUE TITLE
Can't type anything in VSCode

### ISSUE THREAD

**Issue by masaok** [VSCodeVim/Vim#8176]
**Describe the bug**

I just updated to VSCode 1.74.2 with `vscodevim` v1.24.3 and I am unable to type anything.

When I disable the `vscodevim` extension, then I am able to type again.

This is similar behavior to this StackOverflow post: https://stackoverflow.com/questions/45736973/unable-to-type-in-visual-studio-code (even though the post is from five years ago)

**To Reproduce**

1.  Upgrade VSCode to the latest v1.74.2
2.  Install vscodevim v1.24.3
3.  Open any file and try to type something
4.  You should not be able to type anything and hitting escape does nothing

**Expected behavior**
I expect VSCode and `vscodevim` to behavior normally and allow me to edit my files.

**Screenshots**

![image](https://user-images.githubusercontent.com/1320083/209549966-ce29e18d-2893-4c44-9fb3-1683b6e09c14.png)

**Environment (please complete the following information):**

- Extension (VsCodeVim) version:
- VSCode version:
- OS: Windows 10  (WSL2 Ubuntu 20.04 LTS -- the VSCode versions are the same for Windows 10 and WSL2)

![image](https://user-images.githubusercontent.com/1320083/209550020-43f02af0-502b-4e60-bb23-6155306ecf8a.png)

**Additional context**

**Comment by jesuistombe** [comment]
I am experiencing the same issue on Ubuntu 22.04.

**Comment by AchrafAsh** [comment]
Have you tried any of these keys to turn on edit mode:
`a`: append after character
`A`: append at the end of the line
`i`: insert where the cursor is

Checkout [this Vim cheatsheet for more shortcuts](https://devhints.io/vim).

**Comment by masaok** [comment]
> Have you tried any of these keys to turn on edit mode: `a`: append after character `A`: append at the end of the line `i`: insert where the cursor is
> 
> Checkout [this Vim cheatsheet for more shortcuts](https://devhints.io/vim).

@AchrafAsh Thank you for the reply, however, none of the VIM commands worked at the time.

However, I did find (what I believe to be) the solution (thank you @jesuistombe for confirming the issue):

In the VSCode project (repo) that I was in, I had this setting `Vimrc > enable` set to `true`:

![image](https://user-images.githubusercontent.com/1320083/209566770-d9ce3c0e-fc49-44fb-aa0a-466678612e75.png)

When I disabled (set to `false`) this setting, it fixed the problem.

I found the solution when I noticed it was only affecting one of my VSCode projects (repos).

FYI: A nice workaround was the `Simple Vim` extension, which is a slightly "watered-down" version of VIM:

![image](https://user-images.githubusercontent.com/1320083/209566861-8fb376fd-b8d7-4535-afbc-78fde8da5003.png)
